performance_test.R

```diff
@@ -0,0 +1,139 @@
+# Challange 2, Autocomplete me
+# Lynn Kaack
+# Performance of the code
+
+library(microbenchmark)
+
+#source("autocomplete.R") # needs to be sourced manually
```

Comment:
You should have separated out the command-line driver from `autocomplete.R` so that it can be sourced automatically into the test files.